### ISSUE TITLE
Failed to get a json / covjson response following the hyrax guide, donwloads fine

### ISSUE THREAD

**Issue by Mikejmnez** [OPENDAP/hyrax_guide#12]
1.  I am looking at the [Appendix 10.C.10](https://opendap.github.io/hyrax_guide/Master_Hyrax_Guide.html#hyrax-json) on JSON responses. And I try to get the `DAP4 w10n JSON Data request` as described there. Namely:

```
datasetURL = http://test.opendap.org/dap/data/nc/coads_climatology.nc
``` 
get the `DAP4 w10n JSON Metadata request` = ```datasetURL+dmr.json``` and I get an error.

You can see the error http://test.opendap.org/dap/data/nc/coads_climatology.nc.dmr.json

2. I also tried to `Get the CoverageJSON` as described on the [New Features of Hyrax 15.2](https://opendap.github.io/hyrax_guide/Master_Hyrax_Guide.html#cov-json), and nothing happens. I can download it by hitting return and the file is a json file (with metadata and data for each variable). But I am not getting a button option itself. Is this option deprecated and therefore should NOT be incorporated into the Guide?

**Comment by ndp-opendap** [comment]
The json responses only work for DAP2 requests, and they are data responses:
- http://test.opendap.org/dap/data/nc/coads_climatology.nc.json

It would be a nice feature to review the json content and implement the appropriate thing for DAP4.

**Comment by ndp-opendap** [comment]
And the CoverageJSON button is only on the DAP2 Data Request Form:
- http://test.opendap.org/dap/data/nc/coads_climatology.nc.html

**Comment by Mikejmnez** [comment]
Thanks for the clarification! I am in the process of updating the Guide and knowing that CoverageJSON is only for DAP2 responses helps a lot!

**Comment by Mikejmnez** [comment]
However, item 1. above is described on the Guide as a working case, but does not work (`dmr.json`). I can download data just fine and it does follow the json (w10n) spec, but on the browser I get the error that you can reproduce on the hyperlink 
> You can see the error http://test.opendap.org/dap/data/nc/coads_climatology.nc.dmr.json

**Comment by ndp-opendap** [comment]
The error:
> `Unable to transmit response, no transmitter for dmr`

Indicates that there is no software in the server that can re-encode the DMR as a JSON response.

I postulate that you'll get the same error:
> `Unable to transmit response, no transmitter for dap`

If you ask for DAP4 json data:
- http://test.opendap.org/dap/data/nc/coads_climatology.nc.dap.json

**Comment by Mikejmnez** [comment]
yup, indeed.

From our conversation, it seems that json meta and data response, works with DAP2 but it seems that not so in DAP4. That clarification needs be added to the Guide :).

**Comment by Mikejmnez** [comment]
closed by #11
